code/datums/gamemode/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -213,6 +213,41 @@
 		return 0
 	return ..()
 
+//////////////////////////////////////////////
+//                                          //
+//         RAMBLER       (MIDROUND)         ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/midround/from_ghosts/rambler
+	name = "soul rambler migration"
+	role_category = /datum/role/rambler
+	enemy_jobs = list("Librarian","Detective", "Chaplain", "Internal Affairs Agent")
+	required_enemies = list(0,0,1,1,2,2,3,3,3,4)
+	required_candidates = 1
+	weight = 1
+	cost = 5
+	requirements = list(5,5,15,15,25,25,55,55,55,75)
+	logo = "rambler-logo"
+
+/datum/dynamic_ruleset/midround/from_ghosts/rambler/acceptable(var/population=0,var/threat=0)
+	if(!mode.executed_rules)
```

Comment:
actually I think this needs to be !mode.executed_rules.len, as an empty list is not null